operator/common.go

```diff
@@ -35,6 +37,11 @@ var EventTCPAddress = "localhost:4150"
 
 var Pgo config.PgoConfig
 
+// ContainerImageOverrides contains a list of container images that are
+// overridden by the RELATED_IMAGE_* environmental variables that can be set by
+// people deploying the Operator
+var ContainerImageOverrides map[string]string
```

Comment:
It looks to me like every reference to this is in this package; other packages use `SetContainerImageOverride()`. Should this variable be unexported?

Reply:
Nope: https://github.com/CrunchyData/postgres-operator/pull/1212/files#diff-5720bacd93133eaff22b85a98af3c162R358